### PR TITLE
moves Docs WG meeting time for summer

### PR DESCRIPTION
**Pull request by acozine**
Per our discussion on [March 23,](https://github.com/ansible/community/issues/579#issuecomment-800606449) changing the UTC time to one hour earlier for the summer. Depending on your local time, this may mean next week's meeting is at the same local time, or one hour earlier local time.

Related to #579 and to #576.

### CODE REVIEW

File: ansible_community_meetings.ics [hunk 1]
```diff
@@ -8,11 +8,11 @@ DURATION:PT1H
 RRULE:FREQ=WEEKLY
 DESCRIPTION:Project:  Ansible Security Automation working group\nChair:  A
  dam Miller (maxamillion)\, Sumit Jaiswal (justjais)\, Abhijeet Kasurde (ak
- asurde)\nDescription:  \nWe discuss everything about Information Security 
+ asurde)\nDescription:  \nWe discuss everything about Information Security
```
Comment (felixfontein) [inline ansible_community_meetings.ics:11]: I wonder whether these trailing spaces are important and need to stay. Looking at the other lines which don't have spaces, and where words are just broken by a line end, I think they need to stay.

Comment (abadger) [reply ansible_community_meetings.ics:11]: I think you're right.  Maybe adding spaces to the beginning of the lines will be easier to maintain as we will know they're intentional (and IDEs are less likely to remove those automatically).

Comment (acozine) [reply ansible_community_meetings.ics:11]: Oof, my editor must have "done me a favor" there - I didn't intend to change the whitespace. Let me fix that.

Comment (abadger) [reply ansible_community_meetings.ics:11]: https://icalendar.org/iCalendar-RFC-5545/3-1-content-lines.html

Comment (felixfontein) [reply ansible_community_meetings.ics:11]: @abadger I thought that this file was auto-generated anyway from the YAML file, so usually nobody should edit it manually.

**Comment by acozine** [comment]
Note to Future Self: next time update the yaml manually, run make in the meetings subdirectory to update the ics files, then check everything in.

File: ansible_community_meetings.ics (hunk 1 above)
Comment (abadger) [reply ansible_community_meetings.ics]: Yep, you are correct.

**Comment by abadger** [comment]
merged, thanks!

**Pull request merged**